refactoring.api/src/org/netbeans/modules/refactoring/api/RefactoringSession.java

```diff
@@ -352,61 +341,34 @@ private void fireProgressListenerStop() {
             progressSupport.fireProgressListenerStop(this);
         }
     }
-    
-    private void setWrappers(Transaction commit, UndoableWrapper wrap) {
-        wrap.setActive(true, this);
-        
-        //        if (!(commit instanceof RefactoringCommit))
-        //            return;
-        //        for (FileObject f:((RefactoringCommit) commit).getModifiedFiles()) {
-        //            Document doc = getDocument(f);
-        //            if (doc!=null)
-        //                doc.putProperty(BaseDocument.UndoableEditWrapper.class, wrap);
-        //        }
-    }
 
-    private void unsetWrappers(Transaction commit, UndoableWrapper wrap) {
-        wrap.setActive(false, null);
-
-        //        setWrappers(commit, null);
-    }
-
-    private Document getDocument(FileObject f) {
-        try {
-            DataObject dob = DataObject.find(f);
-            EditorCookie cookie = dob.getLookup().lookup(EditorCookie.class);
-            if (cookie == null)
-                return null;
-            return cookie.getDocument();
-        } catch (DataObjectNotFoundException ex) {
-            return null;
-        }
-    }
-
-    
     private class ElementsCollection extends AbstractCollection<RefactoringElement> {
+
         @Override
         public Iterator<RefactoringElement> iterator() {
             return new Iterator() {
-                //private final Iterator<RefactoringElementImplementation> inner = internalList.iterator();
+                
```

Comment:
Maybe delete the trailing spaces as well?

Reply:
It's a new anonymous inner class implementation, right? The members can be separated by a newline from the enclosing curly braces. But they don't have to?

Reply:
I'm fine with the blank lines. It just looked like you were deleting end-of-line whitespace, so the "                " spaces could be deleted as well.